card/lib/card/format.rb

```diff
@@ -419,27 +449,26 @@ def expand_main opts
       opts.merge! root.main_opts if root.main_opts
       legacy_main_opts_tweaks! opts
 
-      #opts[:view] ||= :open
       with_inclusion_mode :normal do
         @mainline = true
-        result = wrap_main nest( root.card, opts )
+        result = wrap_main nest(root.card, opts)
         @mainline = false
         result
       end
     end
 
     def legacy_main_opts_tweaks! opts
-      if val=params[:size] and val.present?
+      if (val = params[:size]) && val.present?
         opts[:size] = val.to_sym
       end
 
-      if val=params[:item] and val.present?
+      if (val = params[:item]) && val.present?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.